Event_Display/scripts/Display.py

```diff
@@ -0,0 +1,29 @@
+#DOES NOT WORK
+
+import numpy as np
+
+import ROOT 
+ROOT.gInterpreter.ProcessLine('#include "../src/DetectorChip.cxx"') 
+ROOT.gInterpreter.ProcessLine('#include "../src/DetectorModule.cxx"')
```

Comment:
We can run the script from base directory this way
```suggestion
ROOT.gInterpreter.ProcessLine('#include "Event_Display/src/DetectorChip.cxx"') 
ROOT.gInterpreter.ProcessLine('#include "Event_Display/src/DetectorModule.cxx"')
```